main.js

```diff
@@ -1,54 +1,83 @@
-// var list = [];
+/* eslint-disable */
+var storedBooksList = JSON.parse(localStorage.getItem('books'));
+console.log(storedBooksList);
+class Book {
+    constructor(title, author, index) {
+        this.title = title;
+        this.author = author;
+        this.index = index;
+    }
+}
 
-// fetching the items from local storage
-const storedBooksList = JSON.parse(localStorage.getItem('books'));
-console.log(storedBooksList.length);
+class Books {
+    constructor() {
+        this.list = [];
+    }
 
-// printing the local storage
-/* eslint-disable */
-for (let i = 0; i < storedBooksList.length; i++) {
-    /* eslint-enable */
-  console.log(storedBooksList[i].title, storedBooksList[i].author);
-  const book = document.createElement('ul');
-  book.className = 'book-menu';
-  book.innerHTML = `<li class="book-name list-group-item">${storedBooksList[i].title}</li>
+    add(title, author) {
+        if (Array.isArray(this.list)) {
+            this.list.push(new Book(title, author));
+        }
+    }
+
+    removeItem(index) {
+        if (Array.isArray(this.list)) {
+            const array = this.list;
+            array.splice(index, 1);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="961" alt="Screen Shot 2022-08-11 at 6 45 22" src="https://user-images.githubusercontent.com/82426682/184126318-ef909029-452f-4d40-a843-df929be9ece6.png">